packages/api/src/CasperJsonRpc/informational/info_get_peers.ts

```diff
@@ -0,0 +1,60 @@
+import { Expose, Type } from 'class-transformer';
+import { IsString, ValidateNested } from 'class-validator';
+
+import {
+  BaseJsonRpc,
+  IRpcResult,
+  JsonRpcOptions,
+  RpcResult,
+  ReturnType
+} from '../../BaseJsonRpc';
+import { DTO, ICamelToSnakeCase } from '../../utils';
+
+export class Peer {
+  @Expose({ name: 'node_id' })
+  @IsString()
+  nodeId: string;
+
+  @IsString()
+  address: string;
+}
+
+export type IPeer = ICamelToSnakeCase<DTO<Peer>>;
+
+export class GetPeersResult extends RpcResult {
+  @Type(() => Peer)
+  @ValidateNested({ each: true })
+  peers: Peer[];
+}
+
+export interface IGetPeersResult extends IRpcResult {
+  peers: IPeer[];
+}
+
+export type GetPeersParams = string[];
+
+export type InfoGetPeersReturnTypeMap = {
+  [ReturnType.Raw]: IGetPeersResult;
+  [ReturnType.Parsed]: GetPeersResult;
+};
+
+/**
+ * Returns a list of peers connected to the node
+ * @param baseJsonRPC
+ * @param options
+ * @returns
+ */
+export async function infoGetPeers<T extends keyof InfoGetPeersReturnTypeMap>(
```

Comment:
Is this the function that should also handle calls like
```
infoGetPeers(__rpcUrl__, configuration) 
```
? Or will this be handled in another PR?

Reply:
discussed this with Jan and he agreed to change the syntax like current implementation